zoo_frontend/src/pages/kitchen/prep/kitchen.jsx

```diff
@@ -9,172 +11,151 @@ import NativeSelect from '@material-ui/core/NativeSelect';
 import NextIcon from '@material-ui/icons/NavigateNext';
 import PrevIcon from '@material-ui/icons/NavigateBefore';
 import Paper from '@material-ui/core/Paper';
-import KitchenView from '../../../components/KitchenView';
+import KitchenView from './KitchenView';
 
-// import { hasAccess, Home, Diet } from '../PageAccess';
+import { Kitchen } from '../../PageAccess';
 
 import {
-  DeliveryContainers,
   DietChanges,
   Diets,
   FoodPrepTables,
   PrepNotes,
-  Species,
 } from '../../../api';
 
+
 export default class extends Component {
   static propTypes = {
     // account: PropTypes.object.isRequired,
-    token: PropTypes.string,
+    // token: PropTypes.string,
     classes: PropTypes.object.isRequired,
-    FoodPrepTables: PropTypes.array.isRequired,
+    prepTables: PropTypes.array.isRequired,
     date: PropTypes.string.isRequired,
-    // eslint-disable-next-line react/no-unused-prop-types
-    PrepDiets: PropTypes.array.isRequired,
-    PrepDietsSub: PropTypes.array.isRequired,
+    prepDiets: PropTypes.array.isRequired,
+    prepDietsSub: PropTypes.array.isRequired,
+    prepNotes: PropTypes.array.isRequired,
+    dietChanges: PropTypes.array.isRequired,
+    table: PropTypes.string,
+    diet: PropTypes.string,
   };
 
   static defaultProps = {
-    token: '',
+    // token: '',
+    table: 0,
+    diet: 0,
   };
 
   static async getInitialProps({ query, authToken }) {
-    const serverFoodPrepTablesAPI = new FoodPrepTables(authToken);
-    const serverDietsAPI = new Diets(authToken);
+    const prepTablesAPI = new FoodPrepTables(authToken);
+    const dietsAPI = new Diets(authToken);
+    const dietChangesAPI = new DietChanges(authToken);
+    const prepNotesAPI = new PrepNotes(authToken);
```

Comment:
I'm starting to mark these as 'serverXXAPI' since they can't be used client side. up to you if you want to follow the same pattern.

Reply:
why cant they be used client side?

Reply:
Technically they 'can' be executed on client excerpt from docs below, but shouldn't be relied upon. Thus you should instantiate two copies of the API one for getInitialProps named 'server'-esq and one for client requests.
```
For the initial page load, getInitialProps will execute on the server only. getInitialProps will only be executed on the client when navigating to a different route via the Link component or using the routing APIs.
```

Reply:
I don't know what you are saying... that excerpt literally says it runs on both.

Reply:
Don't expect to be able to access values declared in getInitialProps on the client's browser that are not returned from the function. The documentation states that the only time the client's browser will have any knowledge of what happened in getInitialProps is iff you use the Link component or use the routing APIs. 

To make it clearer in the code-base, prefix with 'server' to make sure they are used only inside this function.